rpc/api/v1/api.proto

```diff
@@ -59,8 +59,10 @@ message PowParamsResponse {
 
 message SubmitRequest {
     // Proof of Work nonce
+    // deprecated - use certificate instead
```

Comment:
Would it make sense to instead have a dedicated endpoint for "submit with certificate"? The original one would take `nonce` and `pow_params` and the new one the certificate.

This way when we retire the old handler we can just return 404 or a similar error when it is called.

Reply:
That would work for sure, but I think it would complicate the client and CF setup w/o real benefit.

Reply:
CF?

Reply:
cloudflare